### PR TITLE
Remove unused config CATEGORIES_SPLIT_DISPLAY

### PR DESCRIPTION
**Pull request by scottcwilson**
Other unused config values: 

CC_CVV_MIN_LENGTH
CUSTOMERS_AUTHORIZATION_PRICES_OFF
EMAIL_FRIENDLY_ERRORS
EMAIL_LINEFEED
MAX_ROW_LISTS_ATTRIBUTES_CONTROLLER

Suggest removal of all but the first one.

### CODE REVIEW

**Comment by drbyte** [comment]
`CC_CVV_MIN_LENGTH` is probably unnecessary because CVVs should always be 3-4 digits. No reason to be shorter or longer based on what the CC industry is doing presently, AFAIK.

**Comment by drbyte** [comment]
`EMAIL_FRIENDLY_ERRORS` - I don't recall why that would have been removed. Maybe it just assumes always on?

`EMAIL_LINEFEED` - largely moot now.

**Comment by scottcwilson** [comment]
You want to merge this and I'll do the 3 you mentioned in another PR?  Or add them to this PR?

File: zc_install/sql/updates/mysql_upgrade_zencart_158.sql [hunk 1]
```diff
@@ -33,6 +33,9 @@
 # Fix greater than sign in query_builder 
 UPDATE query_builder SET query_name = 'Customers Dormant for 3+ months (Subscribers)' WHERE query_id = 3; 
 
+# Remove unwanted defines
```
Comment (drbyte) [inline zc_install/sql/updates/mysql_upgrade_zencart_158.sql:36]: Perhaps "unwanted" is the wrong word? 

```suggestion
# Remove deprecated defines 
```

**Pull request merged**